### PR TITLE
[DOCS] Clarifying TLS steps

### LINKED ISSUE #65245: Confusing steps in internode TLS setup docs
In https://www.elastic.co/guide/en/elasticsearch/reference/current/configuring-tls.html we have:

> 4. Copy the node certificate to the appropriate locations.
>
>   a.  Create a folder in the configuration directory on each Elasticsearch node to contain your security certificates. For example, create a certs folder in the /home/es/config/certs directory.
> > The Elasticsearch configuration directory varies depending on your Elasticsearch installation.
>
>  b.  Copy the node certificates into the certs directory that you created in the previous step.
>  c.  Copy the .p12 keystore file into the Elasticsearch configuration directory. Elasticsearch will fail to start if the keystore file is located anywhere except this directory.
>  d.  For each additional Elastic product that you want to configure, copy the certificates to the relevant configuration directory.

It looks like this exact wording was introduced in https://github.com/elastic/elasticsearch/pull/63334 when we copied some wording from one section to another, but it doesn't quite land right here.

Steps (b) and (c) are saying more or less the same thing with different words. That can confuse the reader because they assume (c) is asking them to do something additional, but they can't work out what it is.  
I suspect (b) is intended to cover node certificates (from step 2) and (c) is intended to cover the HTTP certificate store (from step 3), but that's not clear.  Perhaps the solution is to explicitly refer to those steps? Something like:

> b. On each node, copy the certificates that you created in step 2, into the `certs` directory that you created in the previous step. This will typically be a single ".p12" (PKCS#12) file
>
> c. If you chose to generate HTTP certificates (step 3), then copy the `http.p12` file into the `certs` directory as well.

_I don't like using "typically" in (b), but going into more explanation is hard, and I'm trying to avoid making big changes, since I know @lockewritesdocs is working on an full overhaul of these instructions.

### PR DESCRIPTION
**Pull request by lockewritesdocs**
Provides clarity around which certificates to copy into configuration directories when configuring TLS.

Closes #65245

### CODE REVIEW

File: x-pack/docs/en/security/securing-communications/node-certificates.asciidoc [hunk 1]
```diff
@@ -88,12 +88,33 @@ Use the <<certutil-http,`elasticsearch-certutil http`>> command:
...
 ----
 
 This command guides you through the process of generating the appropriate
 certificates for use in {es} and {kib}. If you created a CA for your cluster,
 you can re-use it by supplying its location when prompted.
+
+NOTE: The first question that the `elasticsearch-certutil` tool prompts you
+with is whether you want to generate a Certificate Signing Request (CSR).
+Answer `y` if you want to sign your own certificates, or `n` if you want to
+sign certificates with a central CA.
```
Comment (tvernum) [inline x-pack/docs/en/security/securing-communications/node-certificates.asciidoc:101]: I'm not sure that this terminology would clarify which choice to make - that is, "sign your own certificates" isn't particularly clear, and could mean "I want to do it as part of this process" (which would actually mean you should pick `n`) or "I handle it myself by sending it to someone in my organisation" (which is `y`).

I would go with something more like this:
```suggestion
Answer `y` if you need to have your certificates signed by someone else
(such as an internal security team, or a commercial certificate authority),
or `n` if you want to sign certificates with a CA that you have direct access to.
```
It feels a bit wordy though, and "direct access" might not be clear enough.

File: x-pack/docs/en/security/securing-communications/node-certificates.asciidoc [hunk 2]
```diff
@@ -105,12 +126,12 @@ your security certificates. For example, create a `certs` folder in the
 NOTE: The <<config-files-location,{es} configuration directory>> varies
 depending on your {es} installation.
 
-.. Copy the node certificates into the `certs` directory that you created in the
-previous step.
+.. On each node, copy the certificate that you created into the `/certs`
+directory. This certificate is typically a single `.p12` (PKCS#12) file, such
+as `elastic-stack-ca.p12`.
```
Comment (tvernum) [inline x-pack/docs/en/security/securing-communications/node-certificates.asciidoc:131]:
```suggestion
as `elastic-certificates.p12`.
```

File: x-pack/docs/en/security/securing-communications/node-certificates.asciidoc (hunk 1 above)
Comment (lockewritesdocs) [reply x-pack/docs/en/security/securing-communications/node-certificates.asciidoc:101]: I like the distinction that you incorporated. I think we can break these ideas into separate sentences:
>Answer `y` if a trusted authority, such as in internal security team or a commercial certificate authority, will sign your certificates. Answer `n` if you have access to a trusted CA and will sign your own certificates.

**Comment by lockewritesdocs** [comment]
@elasticmachine test this please

**Comment by lockewritesdocs** [comment]
@elasticmachine update branch

**Comment by lockewritesdocs** [comment]
@elasticmachine update branch

**Comment by lockewritesdocs** [comment]
@elasticmachine test this please

**Comment by lockewritesdocs** [comment]
@elasticmachine update branch
